src/periscope.ts

```diff
@@ -499,4 +535,17 @@ function closePreviewEditor() {
   }
 }
 
+// grap the bundled ripgrep binary from vscode
+function ripgrepPath(optionsPath?: string) {
+  if(optionsPath?.trim()) {
+    return optionsPath.trim();
+  }
+
+  const rgBinary = /^win/.test(process.platform) ? "rg.exe" : "rg";
 
+  return path.join(
+    vscode.env.appRoot,
```

Comment:
Rather than depend on drilling for the vscode node modules path we can add the [vscode-ripgrep package](https://github.com/microsoft/vscode-ripgrep) as a dependency for the extension and then extract the `rgPath` from the module

```ts
import { rgPath } from 'vscode-ripgrep';
```